mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -384,12 +388,12 @@ def _id_counter(self):
             yield i
             i += 1
 
-    def set_verbose_logging(self, verbose: bool = False):
+    def set_snippet_client_verbose_logging(self, verbose=False):
+        # type: (bool) -> None
```

Comment:
not sure what this means...remove?

Reply:
It's annotation for python 2 capability.
Remove according previous comment